rooms.js

```diff
@@ -1037,6 +1037,9 @@ let BattleRoom = (() => {
 		logData.endType = this.battle.endType;
 		if (!p1rating) logData.ladderError = true;
 		const date = new Date();
+		logData.timestamp = date;
```

Comment:
Okay, but you should probably do the conversion explicitly `new Date().toString()`, so future people who read your code know that it's intentional and not accidental.